ragaai_catalyst/tracers/agentic_tracing/upload/upload_trace_zip_automation.py

```
@@ -0,0 +1,582 @@
+        except requests.exceptions.RequestException as e:
+            print(f"Error while getting presigned url: {e}")
+            return None
+
+    def _put_presigned_url(self, presignedUrl, filename):
+        headers = {
+                "Content-Type": "application/json",
+            }
+
+        if "blob.core.windows.net" in presignedUrl:  # Azure
+            headers["x-ms-blob-type"] = "BlockBlob"
+        print(f"Uploading agentic traces...")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)